indexing-service/src/main/java/org/apache/druid/indexing/compact/CompactionSupervisorSpec.java

```diff
@@ -77,6 +73,11 @@ public String getId()
     return ID_PREFIX + spec.getDataSource();
   }
 
+  public CompactionConfigValidationResult getValidationResult()
```

Comment:
Please confirm that this doesn't get serialized out. If it does, let's add a `@JsonIgnore`.

Reply:
Confirmed that it doesn't.